server/app/models.py

```diff
@@ -733,7 +744,6 @@ class Score(Base):
     score = ndb.IntegerProperty()
     message = ndb.TextProperty() # Plain text
     grader = ndb.KeyProperty(User) # For autograders, the user who authenticated
-    server_time = ndb.DateTimeProperty(auto_now_add=True)
 
```

Comment:
Added. I'm assuming you made the changes specified in the next comment too. I'll re-apply those.